src/server.ts

```diff
@@ -27,7 +26,7 @@ export default async ({ ego, schemas, tags }) => {
     '/graphql',
     bodyParser.json(),
     graphqlHTTP((err, res) => ({
-      schema: graphqlSchema,
+      schema: createSchema(res.req.body.query.replace(/#.*\n/g, "").split("\"")[1]),
```

Comment:
const query = res.req.body.query.replace(/#.*\n/g, "");
const res = /user\(_id:\s*"(\w*)"/gi.exec(query);
const id = (res && res[1]) ? res[1] : null;

Reply:
Done! :) Couldn't use res as the regex match's name as res.body.query thought I was accessing res before it's instantiation, so that's why its name changed in my commit